integration/test/basic/basic_test.go

```diff
@@ -57,3 +58,52 @@ func TestChartOperatorBootstrap(t *testing.T) {
 		t.Fatalf("bad release status, want %q, got %q", expectedStatus, actualStatus)
 	}
 }
+
+// TestChartConfigChartsInstalled checks that the charts for any chartconfig
+// CRs installed in the cluster have been deployed.
+func TestChartConfigChartsInstalled(t *testing.T) {
+	guestNamespace := "giantswarm"
+	logger, err := micrologger.New(micrologger.Config{})
+	if err != nil {
+		t.Fatalf("could not create logger %v", err)
+	}
+
+	guestG8sClient := g.G8sClient()
+	chartConfigList, err := guestG8sClient.CoreV1alpha1().ChartConfigs(guestNamespace).List(metav1.ListOptions{})
+	if err != nil {
+		t.Fatalf("could not get chartconfigs %v", err)
+	}
+
+	if os.Getenv("CLOP_VERSION_BUNDLE_VERSION") == "0.3.0" {
```

Comment:
Is it because previous versions of cluster-operator don't install chart-operator?

If so why not just returning when previous version bundles are encountered?
Otherwise we'll have to change that every time new version bundle is introduced.